apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -31,17 +30,34 @@ describe('Button Accessibility Testing', () => {
   });
 
   it('Button - Validate accessibilityRole is correct', async () => {
-    await expect(await ButtonPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+    await expect(
+      await ButtonPageObject.compareAttribute(ButtonPageObject._primaryComponent, Attribute.AccessibilityRole, BUTTON_A11Y_ROLE),
+    ).toBeTrue();
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Button - Set accessibilityLabel', async () => {
-    await expect(await ButtonPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(BUTTON_ACCESSIBILITY_LABEL_DEPRECATED);
+    await expect(
+      await ButtonPageObject.compareAttribute(
+        ButtonPageObject._primaryComponent,
+        Attribute.AccessibilityLabel,
+        BUTTON_ACCESSIBILITY_LABEL_DEPRECATED,
+      ),
+    ).toBeTrue();
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Button - Do not set accessibilityLabel -> Default to Button label', async () => {
```

Comment:
nit "Do not set accessibilityLabel -> Validate accessibilityLabel defaults to label"